src/data/markdown/translated-guides/en/03 Results output/200 Real-time/00 Prometheus remote write.md

```diff
@@ -31,60 +31,57 @@ All k6 metric types are converted into an equivalent Prometheus remote write typ
 | Counter | Counter                   | `k6_*_total`         |
 | Gauge   | Gauge                     | `k6_*_<unit-suffix>` |
 | Rate    | Gauge                     | `k6_*_rate`          |
-| Trend   | Gauges / Native Histogram | `k6_*_<unit-suffix>` |
+| Trend   | Gauges (default) / [Native Histogram](#prometheus-native-histogram) | `k6_*_<unit-suffix>` |
 
-### Trend metric conversion
+The output maps the metrics into time series with Name labels.
+As much as possible, k6 respects the [naming best practices](https://prometheus.io/docs/practices/naming) that the Prometheus project defines:
+
+* All time series are prefixed with the `k6_` namespace.
+* All time series are suffixed with the base unit of the sample value (if k6 knows what the base unit is).
+* Trends and rates have the relative suffixes, to make them more discoverable.
+
+## Trend metrics and Histograms
```

Comment:
Agree. 

My intention was to show `Histogram` on the TOC, the right sidebar. Because many folks only read the TOC when moving around pages. Do you have a suggestion to achieve this? 

I am ok accepting this suggestion.

Reply:
As suggested,  using `Trend metrics conversions`